gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -84,7 +84,9 @@ public final void runTask() {
     } catch (IOException exception) {
       kitLogger.error("Error in fetching Build timestamps: " + exception.getMessage());
     }
-    run();
+    if (canExecute()) {
+      run();
+    }
```

Comment:
Maybe we can create an issue to implement a log message whenever a goal (Maven) or task (Gradle) is skipped.

Reply:
WDYT?

Reply:
sounds good to me

Reply:
Do you want me to create issue for this task?

Reply:
https://github.com/eclipse/jkube/issues/1142